sdk-api-src/content/schannel/ns-schannel-crypto_settings.md

```diff
@@ -90,10 +90,17 @@ If 0, schannel uses system defaults. Set to 0 if the CNG algorithm implies bit l
 
 ## -remarks
 
+The following constant distinguishes between the different RSA padding modes and can be specified in the `strCngAlgId` field. Either of them can be used or the <a href="/windows/win32/seccng/cng-algorithm-identifiers">CNG algorithm identifier</a>.
```

Comment:
```suggestion
The following constant distinguishes between the different RSA padding modes and can be specified in the `strCngAlgId` field. Either of these modes can be provided instead of the <a href="/windows/win32/seccng/cng-algorithm-identifiers">CNG algorithm identifier</a>.
```

Reply:
Just a suggested tweak to make sure folks are clear on what can be set.